pom.xml

```diff
@@ -152,11 +152,11 @@
             <plugin>
                 <groupId>com.mycila</groupId>
                 <artifactId>license-maven-plugin</artifactId>
-                <configuration combine.self="override">
-                    <excludes>
-                        <exclude>data/</exclude>
-                    </excludes>
-                </configuration>
+                <!--<configuration combine.self="override">-->
```

Comment:
Never seen this before. Anyhow, if you are going to comment it out, you may as well remove it altogether.